app/models/concerns/indexable.rb

```diff
@@ -30,6 +30,21 @@ def create_index
         body:  { settings: settings, mappings: mappings })
     end
 
+    def update_metadata(version)
+      ES.client.index(
+        index: index_name,
+        type: 'metadata',
+        id: 0,
+        body: { version: version, time: DateTime.now.utc })
+    end
+
+    def stored_metadata
+      ES.client.search(
+        index: index_name,
+        type: 'metadata',
+        body: {query: {match: {_id: 0}}})['hits']['hits'][0]['_source'].symbolize_keys rescue {}
```

Comment:
It's not critical that you fix this. I just noticed you using the `some statement rescue default value` pattern a lot and wanted to highlight the fact that it's not best practice. If you don't change this I'll still merge the PR.